.build/docker/.env.base

```diff
@@ -1,11 +0,0 @@
-VITE_APP_VERSION=0.74.0
```

Comment:
it looks like you're deleting the file. this is not what we want

Reply:
Yes, I need to fix this ebcauses locally `git` isn't telling me it's deleting the file.